package.json

```diff
@@ -9,6 +9,7 @@
   },
   "main": "index.js",
   "scripts": {
+    "install": "npm run build:dll",
```

Comment:
I disagree, the whole point of build:dll is that you shouldn't need to do it ofter.